### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by mrsmkl**
Added simple docker file. Should be able to run the tests. Going to work on adding a testnet next.

### CODE REVIEW

File: contracts/incentive/IncentiveLayer.sol [hunk 1]
```diff
@@ -512,28 +512,27 @@ contract IncentiveLayer is JackpotManager, DepositsManager, RewardsManager {
             s.solution0Challengers.length = 0;
         }
 
-        if (isForcedError(originalRandomBits)) { // this if statement will make this function tricky to test
+        if (isForcedError(t.randomBits, t.blockhash)) { // this if statement will make this function tricky to test
             rewardJackpot(taskID);
-            t.finalityCode = 2;
```
Comment (hswick) [inline contracts/incentive/IncentiveLayer.sol]: Do we want the task to continue as normal if there is a forced error? I thought we were throwing away the task

File: contracts/incentive/IncentiveLayer.sol [hunk 2]
```diff
@@ -512,28 +513,27 @@ contract IncentiveLayer is JackpotManager, DepositsManager, RewardsManager {
...
+        t.state = State.SolutionRevealed;
+        t.randomBits = originalRandomBits;
+        emit SolutionRevealed(taskID, originalRandomBits);
         t.lastBlock = block.number;
     }
 
-    function isForcedError(uint randomBits) internal view returns (bool) {
-        return (uint(keccak256(abi.encodePacked(randomBits, blockhash(block.number)))) < forcedErrorThreshold);
+    function isForcedError(uint randomBits, bytes32 bh) internal view returns (bool) {
+        return (uint(keccak256(abi.encodePacked(randomBits, bh)))%1000000 < forcedErrorThreshold);
```
Comment (hswick) [inline contracts/incentive/IncentiveLayer.sol]: Great! I'm glad you fixed this.

File: contracts/incentive/IncentiveLayer.sol (hunk 1 above)
Comment (mrsmkl) [reply contracts/incentive/IncentiveLayer.sol]: Not sure. But it is more simple if there is no special path. And I cannot see any reason to throw it away.

**Comment by hswick** [comment]
Do you think this is ready to be merged?

**Comment by mrsmkl** [comment]
Yeah, I think it can be merged now.

**Pull request merged**